gerrymandering/gerrymandering.ipynb

```diff
@@ -97,16 +92,15 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "df = dd.io.parquet.read_parquet('./data/census.snappy.parq', engine='pyarrow')\n",
-    "df = df.persist()\n",
-    "census_points = hv.Points(df, kdims=['easting', 'northing'], vdims=['race'])"
+    "df = dd.read_parquet('./data/census.snappy.parq', engine='fastparquet')\n",
+    "df = df.persist()"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Now we can datashade and render these points, coloring the points by race:"
+    "Now we will use hvplot with `datashade=True` to render these points efficiently using [Datashader](http://datashader.org). We also set `dynspread=True` which dynamically increases point size once zooming in enough that that it makes sense to focus on individual points rather than the overall distribution. We also add a tile-based map in the background for context, using `tiles=`."
```

Comment:
```suggestion
    "Now we will use hvplot with `datashade=True` to render these points efficiently using [Datashader](https://datashader.org). We also set `dynspread=True` which dynamically increases point size once zooming in enough that that it makes sense to focus on individual points rather than the overall distribution. We also add a tile-based map in the background for context, using `tiles=`."
```